website/client/src/components/achievements/chooseClass.vue

```diff
@@ -121,44 +121,48 @@
   }
 
   .class-explanation {
-    font-size: 16px;
-    margin: 1.5em auto;
+    font-size: 1rem;
+    margin: 24px auto;
   }
 
   #classOptOutBtn {
     cursor: pointer;
   }
 
   .class-name {
-    font-size: 24px;
+    font-size: 1.5rem;
     font-weight: bold;
-    margin: auto 0.33333em;
+    margin: auto 5px;
   }
 
   .danger {
     color: $red-50;
-    margin-bottom: 0em;
+    margin-bottom: 0;
```

Comment:
Anywhere you're setting a `margin-bottom: 0;` you can use the `mb-0` helper class in the html, will save you some CSS lines

Reply:
Updated!